extension.driver.php

```diff
@@ -137,6 +138,13 @@ public function update($previousVersion=false) {
 				$this->updateAddColumn('text_handle', "ENUM('yes', 'no') DEFAULT 'no' AFTER `text_cdata`");
 			}
 
+			// is handle unique:
+			if (!$this->updateHasColumn('handle_unique')) {
+				$this->updateAddColumn('handle_unique', "ENUM('yes', 'no') DEFAULT 'yes' AFTER `text_handle`");
+				//set default value
+				Symphony::Database()->update(array('handle_unique'=>'yes'),self::FIELD_TABLE);
```

Comment:
Missing space after the comma :P But this line is also not needed as the default value will kicked in if you add NOT NULL (which it should never be!)